remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -308,6 +314,31 @@ do
     fi
 done
 
+# Install and setup the Sumo Collector
+echo "-->Install SumoLogic collector"
+mkdir /tmp/sumo
```

Comment:
Rather than creating multiple temp directories, it might be better to create a temp directory once using `mktemp -d` and then use that as a working directory. It would also ensure that the temporary directory is uniquely named and secured (i.e. accessible only by the current user).

Reply:
I would like to separate different component to different folder. It would be easier to change the code in the future

Reply:
They can still be in subfolders -- they would just be inside the same top-level directory created by `mktemp`.

Reply:
I'll create separate issue if we want to enhance this since this code is from CM deployment shell script